awx/main/tasks/system.py

```diff
@@ -381,16 +376,25 @@ def cleanup_images_and_files():
 
 @task(queue=get_task_queuename)
 def cleanup_host_metrics():
+    """TODO: move to host_metrics in follow-up PR
```

Comment:
nit: Do not use docstring for the TODO. Use a docstring to describe the method on the first line, adding the TODO somewhere in the body of the docstring OR use a normal comment. Consistency, too.